backend/file-ucloud-service/api/src/commonMain/kotlin/QueryDescriptions.kt

```diff
@@ -0,0 +1,274 @@
+package dk.sdu.cloud.file.ucloud.api
```

Comment:
Is this even in use?

Reply:
QueryDescriptions endpoint is not (and is now deleted) but the FileQuery etc. is used in scan.